### DISCUSSION TITLE
Fragmentation of PROTAC molecules

### DISCUSSION THREAD

**Discussion by JIBSN** [rdkit/rdkit#6585] [Q&A]
Hi all,

I want to build a PROTAC database, and currently, I have a large set of structurally diverse PROTAC molecules from patents. I intend to use RDKit to fragment them into warhead, linker, and ligand components. I have worked for it a few days but little progress. I mainly tried substrucuture match, but it's no easy to find the correct bond to fragment on. I am seeking guidance from anyone who has experience with similar fragmentation tasks. Any assistance or advice would be appreciated.

**Comment by shortydutchie** [comment]
I used to do this in two steps, first match a standard set of ligase binders; then match a known set of PoI binders; then identify the linker fragment from the rest after cutting away the two matches.

There is often a bit of ambiguity where exactly the linker starts and ends between PoI and ligase binder - this requires some human guidance. Hence, I had pre-defined sets of smarts to match eg. common CRBN binder motifs, and also the known PoI binders.

There are more fuzzy ways to do it, eg. train a model to recognise these parts - but it won't work well on novel structural classes.
